src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,61 @@
-export const Tabs = () => {};
+import React from 'react';
+import classNames from 'classnames';
+
+import { Tab } from '../../types/Tab';
+
+type Props = {
+  tabs: Tab[];
+  selectedTabId: string;
+  onTabSelected: (tab: Tab) => void;
+};
+
+export const Tabs: React.FC<Props> = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const selectedTab = tabs
+    .find(tab => tab.id === selectedTabId) || tabs[0];
```

Comment:
you have the same logic in App.tsx create a helper function to find selectedTab

Reply:
moved to reusable helper-function